### ISSUE TITLE
Workspace Trust needs accessibility love

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#121094]
For the workspace trust page to be fully accessible it would be nice if we had the following:
- [ ] `up / down` being able to navigate through the whole page
- [x] Whole editor has an aria label which quickly summarises the status. So this would be read once the user open the Workspace Trust UI 
- [ ] We decide what elements should be tabbable (thus they have `tabindex=0` and a nice `aria-label`)
- [x] Status bar entry should have the state in the aria-label. For example currently the "unspecified" and "untrusted" status bar entries have the aria label "Workspace Trust". Instead it should be "Unspecified Workspace Trust" "Untrusted Workspace" or something like that

**Comment by sbatten** [comment]
ref'd wrong issue in commit https://github.com/microsoft/vscode/commit/82767cc1d7bf8cdea0f2897276d5d15aee91f3d9

**Comment by sbatten** [comment]
@isidorn I've pushed changes for the checked items. I'm not exactly sure what the correct flow should be for the remaining items. e.g. What should be focusable and what should use arrow keys vs tabs. I'm happy to implement what is necessary, do you have guidance? I looked at settings and the extension editor for inspiration, but I can't say they were much help.

**Comment by isidorn** [comment]
@sbatten thanks for pushing those changes, I think it improves the experience.
As for tab order here's how I think this would ideally work:

- [x] When the focus is on the whole editor the aria label description is nicely read out
- [ ] If a user press DOWN we move focus to `In a trusted folder box` and all of that box gets read. User can now press tab to focus on the Trust button 
- [ ] If a user press DOWN again we move focus to `In Restricted Mode`
- [ ] If a user press DOWN again we move focus to Trusted Folder list. User can press TAB to move focus into it

Having links as a tab stop IMHO does not make sense. Settings is also not doing that.

As for pressing TAB, I think the only tab stops should be the `In a Trusted Folder` box, `In Restricted Mode` box and the `Trusted Folders` list. So pressing the 4th TAB would take you out of the editor.

The whole point of these changes is that there are not too many tabstops and that there is intuitive navigation in the trust page from top to bottom.
Let me know what you think

**Comment by sbatten** [comment]
@isidorn I have addressed the items above. I did remove tab stops at links though I'm not sure how I feel about it. I feel that the arrows already skip over the links so having them as tab stops is not bad. If I were to revert something, it would be that.

**Comment by isidorn** [comment]
I'll try it out and let you know. Thanks!
